src/uct/ib/mlx5/dc/dc_mlx5.c

```diff
@@ -1612,6 +1612,12 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
                     !!(md->flags & UCT_IB_MLX5_MD_FLAG_DP_ORDERING_FORCE),
                     &config->rc_mlx5_common.super);
 
+    if (uct_ib_iface_is_roce(&self->super.super.super) &&
+        (self->super.super.super.config.dp_ordering_ooo == UCS_TERNARY_LAST)) {
+        ucs_error("Can't set RoCE DP ordering mode");
```

Comment:
1. smallcase
2. print print the device name and requested user config

Reply:
fixed